configuration/modules.yaml

```diff
@@ -70,6 +70,7 @@ modules:
       'cfdplusplus',
       'cmake%gcc@4.8.5',
       'comsol',
+      'crystal',
```

Comment:
The package name is `crystal17` ;-)

Reply:
Yeah... initially I had named the package crystal and then changed to crystal17 and I think all things got mixed up in my head. Sorry about that and thanks for noticing it.